conjur/version.py

```diff
@@ -7,4 +7,4 @@
 is intended to be reused in multiple places.
 """
 
-__version__ = '0.1.1'
+__version__ = '7.0.0'
```

Comment:
This does not match the changelog description (6.0.0)

Reply:
There was some back and forth on versioning. Great catch 💯